NEWS.md

```diff
@@ -8,6 +8,9 @@
 
 ## Mar 2022
 
+* Can now register methods for double-dispatch base Ops (currently only
+  works if both classes are R7) (#128).
```

Comment:
I think it also works if one of the sides is R7 and the other side doesn't have a method for the corresponding op? Like `<R7> + <double>`? It just won't work for `<R7> + <Date>` because `+.Date` exists?

i.e.

``` r
foo <- new_class("foo")

foo() + 1
#> Error: Can't find method for generic `+()` with dispatch classes:
#> - x: foo, R7_object, ANY
#> - y: double, ANY

method(`+`, list(foo, class_double)) <- function(x, y) "foobar"

foo() + 1
#> [1] "foobar"
```

vs

``` r
foo <- new_class("foo")

foo() + Sys.Date()
#> Warning: Incompatible methods ("Ops.R7_object", "+.Date") for "+"
#> Error in foo() + Sys.Date(): non-numeric argument to binary operator

method(`+`, list(foo, new_S3_class("Date"))) <- function(x, y) "foobar"

foo() + Sys.Date()
#> Warning: Incompatible methods ("Ops.R7_object", "+.Date") for "+"
#> Error in foo() + Sys.Date(): non-numeric argument to binary operator
```

Reply:
Oh yeah, good point.